base/event.jl

```diff
@@ -9,15 +89,26 @@ Create an edge-triggered event source that tasks can wait for. Tasks that call [
 `Condition` are suspended and queued. Tasks are woken up when [`notify`](@ref) is later called on
 the `Condition`. Edge triggering means that only tasks waiting at the time [`notify`](@ref) is
 called can be woken up. For level-triggered notifications, you must keep extra state to keep
-track of whether a notification has happened. The [`Channel`](@ref) type does
-this, and so can be used for level-triggered events.
+track of whether a notification has happened. The [`Channel`](@ref) and [`Event`](@ref) types do
+this, and can be used for level-triggered events.
+
+This object is NOT thread-safe. See [`Threads.Condition`](@ref) for a thread-safe version.
 """
```

Comment:
Also, would it be possible to move this docstring down to the definition of `const Condition = GenericCondition{AlwaysLockedST}` (here: https://github.com/JuliaLang/julia/pull/30061/files#diff-b6ceb5f63b0e1cae6694e974c0eaca94R188)?

It's a bit confusing to have it here, i think, because the claim `This object is NOT thread-safe.` depends on which constructor is called.